playwright/tests/rex-test/rex.behaviorspec.ts

```diff
@@ -55,3 +57,29 @@ test('S487 C651123 open keyboard shortcut modal using hot keys', async ({ page }
   await expect(ksModal.ksModal).toBeHidden()
   await expect(page).toHaveURL('/books/organizational-behavior/pages/preface')
 })
+
+test('signup and highlight', async ({ page, isMobile }) => {
+  test.skip(isMobile, 'test only desktop resolution')
+
+  // GIVEN: Open Rex page
+  const BookPage = new ContentPage(page)
+  const path = '/books/introduction-intellectual-property/pages/1-5-what-the-u-s-patent-system-wrought'
+  await BookPage.open(path)
+
+  // AND: Signup as a new user
+  await rexUserSignup(page)
+  await expect(page).toHaveURL('/books/introduction-intellectual-property/pages/1-5-what-the-u-s-patent-system-wrought')
+
+  // WHEN: Highlight some text
+  await BookPage.highlightText()
+
+  // THEN:
+  console.log(BookPage.highlightCount())
```

Comment:
@gregfitch I am trying to count the number of highlights made in a page but seems like I am doing something wrong. The log in line 77 gives `promise pending` instead of 1. Can you please help